src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
 // write your code here
+const logo = document.getElementsByClassName('logo')[0];
+const message = document.createElement('div');
+
+message.className = 'message';
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+Promise.race([promise1, promise2])
+  .then((success) => (message.textContent = success))
```

Comment:
Consider handling the case where `success` might not be a string, although in this context it should always be a string due to the promise resolution.